capistrano-deploy.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
 
+  s.add_dependency('rails', '>= 3.0.0')
```

Comment:
It looks like we're adding `rails` as a dependency to get the `titleize` method?

What do you think about adding ActiveSupport instead?

i.e.

```ruby
# in gemspec
s.add_dependency 'activesupport'

# in lib/capistrano-deploy.rb
require 'active_support/all'
```